### PR TITLE
Issue 793 - English docstring of each chart.

### PR DESCRIPTION
**Pull request by sunhailin-Leo**
* Fix issue 793 https://github.com/pyecharts/pyecharts/issues/793

### CODE REVIEW

**Comment by chenjiandongx** [comment]
@chfw 你的领域 出来评价评价

File: pyecharts/charts/basic_charts/effectscatter.py [hunk 1]
```diff
@@ -6,9 +6,10 @@
 
 class EffectScatter(RectChart):
     """
-    <<< 带有涟漪特效动画的散点图 >>>
+    <<< Scatter plots with ripple effects animation >>>
 
-    利用动画特效可以将某些想要突出的数据进行视觉突出。
+    Use animation effects to visually highlight
+    certain data that you want to highlight.
```
Comment (chfw) [inline pyecharts/charts/basic_charts/effectscatter.py]: Use animation effects to visually highlight 
certain data that you want to highlight. 

->

Use animation effects to visually highlight designated data set.

File: pyecharts/charts/basic_charts/funnel.py [hunk 2]
```diff
@@ -6,7 +6,13 @@
 
 class Funnel(Chart):
     """
-    <<< 漏斗图 >>>
+    <<< Funnel >>>
+
+    Funnel diagram is suitable for one-way analysis of single process
+    with standardized business process, long cycle and multiple links.
+    Through comparison of business data of each link in the funnel,
+    the link where the problem can be found and explained intuitively,
```
Comment (chfw) [inline pyecharts/charts/basic_charts/funnel.py]: potential problems can be found intuitively,

File: pyecharts/charts/basic_charts/treemap.py [hunk 3]
```diff
@@ -6,10 +6,10 @@
 
 class TreeMap(Chart):
     """
-    <<< 树图 >>>
+    <<< TreeMap >>>
 
-    树图是一种常见的表达『层级数据』『树状数据』的可视化形式。它主要用面积的方式，
-    便于突出展现出『树』的各层级中重要的节点。
+    TreeMap are a common visual representation of "hierarchical data" and "tree data".
+    It mainly use area to highlight the important nodes in the hierarchy of "tree".
```
Comment (chfw) [inline pyecharts/charts/basic_charts/treemap.py]: use -> uses

File: pyecharts/charts/basic_charts/bar.py [hunk 4]
```diff
@@ -6,9 +6,9 @@
 
 class Bar(RectChart):
     """
-    <<< 柱状图/条形图 >>>
+    <<< Bar Chart >>>
 
-    柱状/条形图，通过柱形的高度/条形的宽度来表现数据的大小。
+    Bar chart, which indicates the size of the data by the height/width of the bar.
```
Comment (chfw) [inline pyecharts/charts/basic_charts/bar.py]: Bar chart  presents categorical data with rectangular bars with heights or lengths proportional to the values that they represent

-- from wikipedia :)

File: pyecharts/charts/basic_charts/gauge.py [hunk 5]
```diff
@@ -6,7 +6,9 @@
 
 class Gauge(Chart):
     """
-    <<< 仪表盘 >>>
+    <<< Gauge >>>
+
+    The gauge measures information and key business metrics
```
Comment (chfw) [inline pyecharts/charts/basic_charts/gauge.py]: The gauge displays key business measure.

File: pyecharts/charts/basic_charts/graph.py [hunk 6]
```diff
@@ -6,9 +6,9 @@
 
 class Graph(Chart):
     """
-    <<< 关系图 >>>
+    <<< Graph >>>
 
-    用于展现节点以及节点之间的关系数据。
+    The graph is used to represent nodes and the relational data between them.
```
Comment (chfw) [inline pyecharts/charts/basic_charts/graph.py]: The graph is used to represent the relational data.

**Comment by chfw** [comment]
tried my best with my Chinglish knowledge

**Comment by chfw** [comment]
related to https://github.com/pyecharts/pyecharts/issues/965

**Pull request merged**